besu/src/main/java/org/hyperledger/besu/cli/subcommands/networkcreate/generate/DirectoryHandler.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package org.hyperledger.besu.cli.subcommands.networkcreate.generate;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static java.nio.file.Files.createDirectory;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Objects;
+// TODO Handle errors
+public class DirectoryHandler {
+
+  public void create(final Path outputDirectoryPath) {
+    checkNotNull(outputDirectoryPath);
+    final File outputDirectory = outputDirectoryPath.toFile();
+
+    if (outputDirectory.exists()
```

Comment:
(suggestion): consider using `Files.isDirectory` (https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#isDirectory(java.nio.file.Path,%20java.nio.file.LinkOption...))
This method returns true if the file is a directory; false if the file does not exist, is not a directory, or it cannot be determined if the file is a directory or not.  So you can remove unnecessary check (`exists`).